src/components/TranslationProvider.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactElement, useEffect, useState } from "react";
+import { TranslationContext } from "../contexts/TranslationContext";
+import { LANGS } from "../types/TranslationTypes";
+import defaultLang from "../i18n/en.json";
+
+export default function TranslationProvider(props: { children: ReactElement }) {
```

Comment:
It might be not clear from an issue itself, but the idea was not only to set up base for i18n, but also to implement language switcher - currently the language is always default in fact

Reply:
possibility to change language already implemented, just call `setCurrentLang(lang)` to change config.
for ex.: 
<img width="460" alt="Screenshot 2021-09-07 at 11 49 56" src="https://user-images.githubusercontent.com/33167093/132326584-f7d0c7b6-4431-467b-8af1-cdb96a3feb82.png">

if we speak about UI elements I plan to add buttons after adding material-ui, so as not to rewrite them